[Add] pos-platform_order_gofood: new doc (#17452)
odoo/documentation: content/applications/sales/point_of_sale/restaurant/APAC_food_delivery.rst

==================
APAC food delivery
==================

Point of Sale provides integrations with popular APAC food delivery platforms, allowing for

> synchronizing orders, updating menus and managing store details directly from the :guilabel:`POS`.

Reviewer: `:guilabel: ` is used for on-screen text, like a menu, setting, etc. [See here](https://www.odoo.com/documentation/19.0/contributing/documentation/rst_guidelines.html#gui-element) + I added a coma
```suggestion
Point of Sale provides integrations with popular APAC food delivery platforms, allowing for
synchronizing orders, updating menus, and managing store details directly from the POS.
```

Supported providers:

- :ref:`GoFood <pos/apac/gofood>`
- GrabFood

.. important::
   A valid account with each delivery provider is required before proceeding.

.. _pos/apac/general configuration:

General configuration
=====================

> Before setting up a specific delivery provider, configure your general store hours and prepare your products for synchronization.

Reviewer: Always [break the line at the 100th character](https://www.odoo.com/documentation/19.0/contributing/documentation/rst_guidelines.html#th-character-limit)
 
```suggestion
Before setting up a specific delivery provider, configure your general store hours and prepare your
products for synchronization.
```

.. _pos/apac/general_configuration/service_hours:

Service hours
-------------

> Configure the store timings to define when delivery services are available. These hours synchronize directly with the respective delivery platforms.

Reviewer: ```suggestion
Configure the store timings to define when delivery services are available. These hours synchronize
directly with the respective delivery platforms.
```

#. Navigate to :menuselection:`Point of Sale --> Configuration --> Service Hours`.
#. Click :guilabel:`New` to add a record.

> #. Define the :guilabel:`Name`, :guilabel:`Day of Week`, :guilabel:`Opening from`, and :guilabel:`Opening to` associated with this working time.

Reviewer: ```suggestion
#. Define the :guilabel:`Name`, :guilabel:`Day of Week`, :guilabel:`Opening from`, and
:guilabel:`Opening to` associated with this working time.
```
